### PR TITLE
Update payment card spec

### PR DESCRIPTION
**Pull request by dara123**

### CODE REVIEW

File: spec/paths/payment-cards@{id}.yaml [hunk 1]
```diff
@@ -39,9 +39,25 @@ patch:
           cvv:
             description: Card's cvv (card verification value).
             type: string
+            required: false
+          expMonth:
+            description: Card's expired month
```
Comment (adamaltman) [inline spec/paths/payment-cards@{id}.yaml]: Card's expiry month

File: spec/paths/payment-cards@{id}.yaml [hunk 2]
```diff
@@ -39,9 +39,25 @@ patch:
...
+            type: string
+            required: false
+          expYear:
+            description: Card's expired year
```
Comment (adamaltman) [inline spec/paths/payment-cards@{id}.yaml]: Card's expiry year

File: spec/paths/payment-cards@{id}.yaml [hunk 3]
```diff
@@ -39,9 +39,25 @@ patch:
           cvv:
             description: Card's cvv (card verification value).
             type: string
+            required: false
```
Comment (adamaltman) [inline spec/paths/payment-cards@{id}.yaml]: required is false by default, so why include it here?

Comment (slavcodev) [reply spec/paths/payment-cards@{id}.yaml]: actually is not supported by swagger, `required` is attribute of the `object` that contains list of required properties, is not a flag for every field

Comment (adamaltman) [reply spec/paths/payment-cards@{id}.yaml]: Yes, I think this is why the tests are failing.

File: spec/paths/payment-cards.yaml [hunk 4]
```diff
@@ -43,6 +43,17 @@ post:
       schema:
         $ref: "#/definitions/PaymentCard"
   responses:
+    200:
+      description: PaymentCard was updated
```
Comment (adamaltman) [inline spec/paths/payment-cards.yaml]: isn't this created?

Comment (adamaltman) [reply spec/paths/payment-cards.yaml]: oh nevermind, I understand.

**Pull request merged**